test/integration/crud_test.go

```diff
@@ -20,22 +20,21 @@ import (
 	"fmt"
 	"testing"
 
-	apiv1 "k8s.io/api/core/v1"
-
-	"k8s.io/api/core/v1"
-
-	"k8s.io/apiserver/pkg/storage/names"
-
 	"github.com/pborman/uuid"
 
+	"k8s.io/api/core/v1"
+	apiv1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	pkgruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/storage/names"
 	federationapi "k8s.io/federation/apis/federation/v1beta1"
 	"k8s.io/federation/pkg/federatedtypes"
 	"k8s.io/federation/pkg/federatedtypes/crudtester"
 	"k8s.io/federation/test/integration/framework"
 )
 
+// This test is intended to allow api-only tests to run in both integration and e2e
```

Comment:
I'm assuming this is intended for the follow-on PR?

Reply:
yes, I realized this was not at all accurate when I was working on the follow-on PR. I can change it to reflect the current state of things.